src/components/Button/FallbackButton.js

```diff
@@ -5,7 +5,7 @@ import style from './style.scss'
 const FallbackButton = ({ text, onClick }) => (
   <button
     type="button"
-    className={`${style.fallbackButton} ${theme.warning}`}
+    className={`${style.fallbackButton} ${theme.warningFallbackButton}`}
```

Comment:
Just a minor change:

```suggestion
    className={classNames(style.fallbackButton, theme.warningFallbackButton)}
```